doc/use_cases.md

```diff
@@ -46,22 +52,38 @@ For each jugement object returned by:
 
     curl http://ctiahost/ctia/ip/192.168.1.1/indicators
 
-Extract the IDs from the `related_campaigns.campaign__id' fields and
+Extract the IDs from the 'related_campaigns.campaign_id' fields and
 
     curl http://ctiahost/ctia/campaigns/ID
 
-### 2.3 As an incident responder, I would like to know what suggest COAs are asociated with an IP
+### 2.3 As an incident responder, I would like to know what suggested COAs are asociated with an IP
 
-For each jugement object returned by:
+For each indicator object returned by:
 
     curl http://ctiahost/ctia/ip/192.168.1.1/indicators
 
-Extract the IDs from the `suggested_COA.COA__id' fields and
+Extract the IDs from the 'suggested_COA.COA__id' fields and
 
     curl http://ctiahost/ctia/coa/ID
 
 ### 2.4 As an incident responder, I would like to record an incident and it's sightings
 
+Determine the ID of the incident's indicator(s).  For example, if you know the title:
+
+	curl http://ctiahost/ctia/indicator/title/document-direct-ip-traffic
+
+Or search for the indicator:
+
+    curl http://ctiahost/ctia/indicator/_search?title=ip # URL TBD
```

Comment:
By proposing to use POST for a read-only method, I somehow break the REST principe. By using the body in a GET request we would apparently don't follow the HTTP1.1 recommandation to ignore the body on GET queries (while I don't think this is perfectly clear).

So I don't really like to use POST because it breaks a nice REST specification and also because it won't be natural for users.

Do you think that starting to use GET with standard query parameters and then change it only if we encounter limitations would be the right choice?